distribution/lib/Standard/AWS/0.0.0-dev/src/S3/S3_Data_Link.enso

```diff
@@ -19,7 +19,7 @@ type S3_Data_Link
     Value (uri : Text) format_json (credentials : AWS_Credential)
 
     ## PRIVATE
-    parse json -> S3_Data_Link =
+    parse json _ -> S3_Data_Link =
```

Comment:
I know we are discarding this but can we do:
```suggestion
    parse json source -> S3_Data_Link =
        _ = source
```
as clearer to read.

Reply:
Ok sure, I was also not sure if `_` is right here, I think that named argument is indeed better - will amend all of them